proto/tabletmanagerdata.proto

```diff
@@ -831,8 +829,7 @@ enum CheckThrottlerResponseCode {
 }
 
 message CheckThrottlerResponse {
-  // StatusCode is HTTP compliant response code (e.g. 200 for OK)
-  int32 status_code = 1;
+  reserved 1; // was `status_code`, HTTP compliant, deprecated
```

Comment:
Do we care about reserving the name too? I'm not sure, just asking. We can always do that later since your comment clearly indicates what the name was. It's also fine to eventually reuse field names, as long the json/text encodings are still usable and sensible.

Reply:
Added reserved names.